jobs/kpi-forecasting/todo-dag.py

```diff
@@ -0,0 +1,36 @@
+"""
```

Comment:
This can be removed. I created a DAG here: https://github.com/mozilla/telemetry-airflow